placekey/sf/doc/REFERENCE.md

```diff
@@ -0,0 +1,104 @@
+## placekey
+
+[Placekey](https://www.placekey.io/faq) is a free and open universal standard identifier for any physical place, so that the data pertaining to those places can be shared across organizations easily. Since Placekey is based on H3, here we offer a way to transform to and from that index and delegate any extra functionality to H3 itself.
+
+You can learn more about Placekey on [their website](https://www.placekey.io/) or in the [Overview section](../../overview/spatial-indexes/#placekey) of this documentation.
+
+### H3_ASPLACEKEY
+
+{{% bannerNote type="code" %}}
+PLACEKEY.H3_ASPLACEKEY(h3index)
```

Comment:
I would use lowercase for the module name to be aligned with the BigQuery reference.

placekey.H3_ASPLACEKEY(h3index)